README.md

```diff
@@ -149,7 +149,7 @@ token with no extra permissions is enough to be able to check public repos links
 There is an extensive list of commandline parameters to customize the behavior,
 see below for a full list.
 
-```
+```sh
```

Comment:
I think this is causing weird syntax highlight when it's rendered: https://github.com/lycheeverse/lychee#commandline-parameters

Reply:
true, let me fix it

Reply:
done in `master`.

Reply:
Awesome 😎